core/tests/test_utils.py

```diff
@@ -251,6 +282,7 @@ def signup(self, email, username):
         """Complete the signup process for the user with the given username."""
         self.login(email)
 
+        self.enable_urlfetch_mock()
```

Comment:
Ah, I see. Thanks for explaining! This is pretty important context, so could you add a comment here?

Also, I think this would be better done using a context manager (i.e. using 'with') as mentioned above.